src/Symfony/Component/Validator/ExecutionContext.php

```diff
@@ -92,6 +92,12 @@ public function __construct(GlobalExecutionContextInterface $globalContext, Tran
      */
     public function addViolation($message, array $params = array(), $invalidValue = null, $plural = null, $code = null)
     {
+        trigger_error(
+            'The parameters $invalidValue, $pluralization and $code are not supported anymore '.
+            'as of Symfony 2.5 and will be removed in Symfony 3.0. Please use buildViolation() instead.',
```

Comment:
`are deprecated since version 2.5 and will be removed 3.0` to keep the same template everywhere please